spec/src/main/asciidoc/ch06-criteria-api.adoc

```diff
@@ -80,9 +80,9 @@ this specification.]
 managed class by appending "pass:[_]" to the name of the managed class.
 * The metamodel class _pass:[X_]_ must be annotated with the
 _jakarta.persistence.StaticMetamodel_ annotationfootnote:[If the class
-was generated, the _jakarta.annotation.Generated_ annotation should be
-used to annotate the class. The use of any annotations other than
-_StaticMetamodel_ and _Generated_ on static metamodel classes is
+was generated, the _javax.annotation.processing.Generated_ or _jakarta.annotation.Generated_
+annotation should be used to annotate the class. The use of any annotations other than
+_StaticMetamodel_ and any of _Generated_ on static metamodel classes is
```

Comment:
"any of" -> "either"?

or just:

> The use of any other annotations on static metamodel classes is undefined.

Reply:
the rewrite is better (and shorter) => done